.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - track/**
+
+jobs:
+  test-charm:
+    uses: ./.github/workflows/integration_tests.yaml
+
+  release-charm:
+    name: Release the charm
+    needs:
+      - ci-tests
```

Comment:
just to confirm: should this be `test-charm` instead?

Reply:
yeh, you are right, I renamed it and forgot to update it here.